apps/nextjs/src/app/(main)/organizations/[org_slug]/page.tsx

```
@@ -37,21 +37,33 @@ async function OrganizationContent({ org_slug }: { org_slug: string }) {
+          organization={organization}
+          classNames={{ wrapper: "w-11/12 my-4" }}
+        >
+          <div className="mx-4 flex h-full w-full flex-col items-center justify-between py-2 text-center">
+            <h1 className="text-2xl font-semibold">{organization.name}</h1>
+            <p>{organization.description}</p>
+          </div>
+        </OrganizationHeader>
+      </section>
+
+      <section
+        role="region"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<section>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)